components/NewOutivityForm/index.js

```diff
@@ -0,0 +1,155 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function NewOutivityForm({
+  CreateOutivity,
+  handleCancel,
+  selectedImage,
+}) {
+  return (
+    <main>
+      <StyledNewOutivitiesForm onSubmit={CreateOutivity}>
+        <h1>New Outivity</h1>
+        <StyledNewOutivitiesFormFields>
+          <StyledNewOutivitiesFormField>
+            <label htmlFor="image">Image</label>
+            {selectedImage && (
+              <Image
+                src="/{selectedImage}"
```

Comment:
selectedImage is a JS Variable, if you were to pass it as the src as you did previously, it would be always be the string "/{selectedImage}". If you want to have access to what is stored in the selectedImage variable, you need to leave out the quotation marks and the slash.

```suggestion
                src={selectedImage}
```